operator/roles/ansible-service-broker/templates/broker.deployment.yaml.j2

```diff
@@ -59,6 +59,12 @@ spec:
           env:
             - name: BROKER_CONFIG
               value: /etc/{{ broker_name }}/config.yaml
+{% for proxy_envvar in ["HTTP_PROXY", "HTTPS_PROXY", "NO_PROXY"] %}
```

Comment:
Will this handle lowercase proxy vars? I know we had to deal with lowercase ones in bundle-lib and when I did subscription-manager.

https://github.com/automationbroker/bundle-lib/blob/master/runtime/run_bundle.go#L225-L236
https://github.com/candlepin/subscription-manager/blob/master/test/rhsm/unit/util_tests.py#L319-L320

maybe they'll always be uppercase in this scenario but I know I had to deal with them possibly being lowercase.

Reply:
They are being passed specifically as uppercase from the contract with OLM